csrankings.csv

```diff
@@ -6619,6 +6636,7 @@ José António Beltran Gerald,Universidade de Lisboa,http://ieeexplore.ieee.org/
 José Augusto Suruagy Monteiro,UFPE,http://www.di.ufpe.br/~suruagy,xCIQf-QAAAAJ
 José Bento,Boston College,http://www.jbento.net,8K6Z0lEAAAAJ
 José Bento Ayres Pereira,Boston College,http://www.jbento.net,8K6Z0lEAAAAJ
+José C. Cunha,Universidade NOVA de Lisboa,http://nova-lincs.di.fct.unl.pt/person/1,VeZ4QbAAAAAJ

```

Comment:
Per his home page, he retired a number of years ago.
http://asc.di.fct.unl.pt/~jcc/